src/pages/soknad/[uuid]/index.tsx

```diff
@@ -77,7 +77,12 @@ export async function getServerSideProps(
   const arbeidsforholdResponse = await getArbeidsforhold(onBehalfOfToken);
 
   if (arbeidsforholdResponse.ok) {
-    arbeidsforhold = await arbeidsforholdResponse.json();
+    const arbeidsforholdList = await arbeidsforholdResponse.json();
+
+    arbeidsforhold = [...arbeidsforholdList].map((forhold: IArbeidsforhold) => ({
```

Comment:
Dette for å sette default `utfyllingStatus` feltet for å kunne sette `Fullført` eller `Påbegynt` status i accordion